Source/WTF/wtf/CheckedPtr.h

```diff
@@ -238,4 +277,4 @@ template<typename T> using PackedCheckedPtr = CheckedPtr<T, PackedPtrTraits<T>>;
 
 using WTF::CheckedPtr;
 using WTF::PackedCheckedPtr;
-
+using WTF::static_pointer_cast;
```

Comment:
I don't think we want to expose this since static_pointer_cast is not safe to use.